packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -158,7 +159,16 @@ export class EpochTracker {
             this.logger.sendErrorEvent({ eventName: "EpochVersionMismatch" }, error);
             assert(!!this._hashedDocumentId, "DocId should be set to clear the cached entries!!");
             // If the epoch mismatches, then clear all entries for such document from cache.
-            await this.persistedCache.removeAllEntriesForDocId(this._hashedDocumentId);
+            const entry: ICacheEntry = {
+                file: {
+                    docId: this._hashedDocumentId,
+                    // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+                    resolvedUrl: {} as IOdspResolvedUrl,
+                },
+                type: "snapshot",
+                key: "",
```

Comment:
From API perspective, it does not make sense to have key here, I'd change signature to only include file & type on that interface that we use here.
Or alternatively, we can say that all fields are optional, and presence of any field means only entries with this field should be deleted, otherwise all entries with any key should be deleted.

Reply:
Included the file entry to be deleted instead of docid.